react/ProductName.tsx

```diff
@@ -21,7 +22,7 @@ const ProductName: FunctionComponent = () => {
       className={`c-on-base t-title lh-copy fw6 no-underline fw5-m ${
         handles.productName
       } ${opaque(item.availability)}`}
-      href={item.detailUrl || undefined}
+      href={rootPath + item.detailUrl || undefined}
```

Comment:
if the `item.detailUrl` doesn't exist we would end up with `<a href="undefined">` which is undesirable

```suggestion
      href={item.detailUrl ? rootPath + item.detailUrl : undefined}
```

Reply:
@lucasecdb Hi! Thanks for the suggestion and makes sense. It's done.

Reply:
@lucasecdb Can I got an example mock for useRuntime? I did one but I'm not sure that's everything ok.